src/python/pants/reporting/zipkin_reporter.py

```diff
@@ -104,7 +104,7 @@ def start_workunit(self, workunit):
         self.parent_id = zipkin_attrs.span_id
 
       span = local_tracer.zipkin_span(
-        service_name=service_name,
+        service_name="pants/corp/{}/local".format(service_name),
```

Comment:
`corp` probably doesn't mean anything to most folks.

I'd suggest maybe adding an option to the runtracker to let people configure the prefix here, and defaulting it something like `pants/`.

Reply:
Zipkin team said that the convention is `service/env/role/zone`, and asked that `zone` to be `local`. And `pants/corp/pants/local` was their suggestion. I saw examples of `env` to be `staging`, `devel`, `prod`.
Response from Zipkin team:
```
Jonathan Simms  [< 1 minute ago]
devel, staging, prod


Jonathan Simms  [< 1 minute ago]
for aurora

Jonathan Simms  [< 1 minute ago]
i believe that's true i.e. corp for pants
```